internal/server/turn.go

```diff
@@ -76,7 +76,7 @@ func handleAllocateRequest(r Request, m *stun.Message) error {
 	// 5.  The server checks if the request contains a RESERVATION-TOKEN
 	//     attribute.  If yes, and the request also contains an EVEN-PORT
 	//     attribute, then the server rejects the request with a 400 (Bad
-	//     Request) error.  Otherwise, it checks to see if the token is
```

Comment:
Seems like an odd change? Why is there a tab?

Reply:
Fixed per [c5dca04](https://github.com/pion/turn/pull/267/commits/c5dca04e772ad98d54da1409afc8803b2d7900de).